go/.tools/gen_config.sh

```diff
@@ -0,0 +1,11 @@
+#/bin/bash
+mkdir -p ~/.paddle
+cat > ~/.paddle/config << EOF
+datacenters:
+- name: datacenter1
+  username: user@baidu.com
+  password: T123
+  endpoint: http://127.0.0.1:8080
+  pfsendpoint: http://127.0.0.1:8080
```

Comment:
It's not simple enough for users to configure if we have two "endpoints", just use static paths like `http://127.0.0.1:8080/api/v1/jobs` and `http://127.0.0.1:8080/api/v1/pfs`, so users only need to configure endpoint to `http://127.0.0.1:8080`